rpm/nemo-qml-plugin-filemanager.spec

```diff
@@ -22,6 +21,8 @@ BuildRequires:  pkgconfig(KF5Archive)
 %package devel
 Summary:    Filemanager C++ library
 Requires:   %{name} = %{version}-%{release}
+Requires:   pkgconfig(Qt5Core)
+Requires:   pkgconfig(Qt5Qml)
```

Comment:
The packageconfig does include those. Specifying them manually is redundant and wrong.

Reply:
Removed.